src/extensions/rich-text/paste-markdown.ts

```diff
@@ -66,27 +69,42 @@ const PasteMarkdown = Extension.create({
                                   ) as VSCodeClipboardMetadata)
                                 : {}
 
+                        const clipboardContainsHTMLFromVSCodeOtherThanTextOrMarkdown =
+                            clipboardContainsVSCodeMetadata &&
+                            // If `mode` from the VS Code metadata is `null` it probably means that
+                            // the user has the VS Code `editor.copyWithSyntaxHighlighting` setting
+                            // set to `false`, thus returning plain-text
+                            vsCodeClipboardMetadata.mode !== null &&
```

Comment:
This is one of the functional changes, which makes pasting content copied from VS Code work in Chromium browsers regardless if users have `editor.copyWithSyntaxHighlighting` set to `true` (default if not set), or `false`.